accepted/2025/mono.md

```diff
@@ -0,0 +1,108 @@
+If nothing is changing in .NET Framework or in Mono, there is very little reason to be running our tests against Mono.
+
+
+## Explanation
+
+### Functional explanation
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+
+We do not officially support Mono which is specified at https://learn.microsoft.com/en-us/nuget/resources/nuget-faq#does-nuget-support-mono- .
+However, we've continued to run our tests on Mono to ensure that we do not unintentionally break functionality.
+
+We would not take any intentional actions to make `NuGet.exe` stop working on Mono, we would just not guarantee that it continues to work.
+We would also continue to test `NuGet.exe` on Windows .NET Framework, so it is unlikely that functionality would break.
+
+We will take the following actions:
+
+1. Stop running tests on any Mono runtime, regardless of operating system.
+1. Add a note to the GitHub action [setup-nuget](https://github.com/NuGet/setup-nuget) that running `NuGet.exe` on Mono is not officially supported.
+1. Update documentation to indicate that installing the Mono runtime on hosted build agents may be required depending on the users' environment since some companies are leaving Mono out of agent images.
+1. Add XML documentation comments to our code in areas where there is Mono specific logic to indicate that Mono is not officially supported but to not remove any existing code.
+1. Consider closing all [existing issues related to Mono](https://github.com/NuGet/Home/issues?q=is%3Aissue%20state%3Aopen%20label%3APlatform%3AMono)
+
+
+### Technical explanation
+<!-- Explain the proposal in sufficient detail with implementation details, interaction models, and clarification of corner cases. -->
+
+There are a few places in our code with [Mono-specific logic](https://github.com/search?q=repo%3ANuGet%2FNuGet.Client%20mono&type=code) and we would not be removing any of this.
+However, we should add XML doc comments to indicate that Mono isn't supported but we should not remove the code either.
+
+To stop running tests on Mono we could simply disable [the parameter](https://github.com/NuGet/NuGet.Client/blob/dev/eng/pipelines/pull_request.yml#L40) by default which would be simpler.
+Alternatively, we could completely remote it from the pipeline YAML.
+Since the functionality could get broken over time, there's no guarantee that leaving the option in place would work going forward.
+
+## Drawbacks/Risks
+<!-- Why should we not do this? -->
+
+NuGet does not want to break Mono functionality or remove anyone's ability to run `NuGet.exe` on Mono.
+However, by no longer running the tests on the Mono runtime, we will not be able to guarantee that we don't unintentionally break something.
```

Comment:
This says something similar to line 35

Reply:
I've updated the wording, let me know what you think